ksql-functional-tests/src/test/resources/query-validation-tests/delimited.json

```diff
@@ -178,7 +178,7 @@
         }
       ],
       "expectedException": {
-        "type": "io.confluent.ksql.util.KsqlStatementException",
+        "type": "io.confluent.ksql.util.KsqlException",
```

Comment:
Switching this means poor UX: if a user submits a request with multiple statements in it, the error doesn't identify _which_ statement had the error.   This can make it hard(er) for the user to track down the cause of the error.  

Better to fix the code to ensure the statement exception is thrown.

Reply:
I tried to do this but it wasn't super straightforward because we no longer have access to the statement itself where the exception is thrown. I was thinking of just wrapping all exceptions thrown from the Analyzer somewhere high up - I will just do that.

Reply:
Since the build is green and I've got a follow-up refactor coming up I'll address these in that one.